test/lightweight-tests/pom.xml

```diff
@@ -547,6 +547,12 @@
             <version>1.12.251</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Please comment this dependency as it is unfortunately not public. It makes the CI fail.